app/models/developers/title.rb

```diff
@@ -0,0 +1,45 @@
+module Developers
+  class Title
+    private attr_reader :query
+
+    def initialize(query)
+      @query = query
+    end
+
+    def title
+      components = []
+      components << "Hire"
+      components << role_level if role_level
+      components << "freelance" if freelance?
+      components << "Ruby on Rails developers"
+      components << "in #{country}" if country
+      components.join(" ")
+    end
```

Comment:
Translating this in the future will be very difficult. How can I make this easier to manage?